nucypher/blockchain/eth/policies.py

```diff
@@ -225,7 +225,9 @@ def make_arrangements(self, network_middleware: RestMiddleware, *args, **kwargs)
         if actual_sample_quantity > 0:
             # Sample by reading from the Blockchain
             try:
-                sampled_addresses = self.alice.recruit(quantity=actual_sample_quantity, duration=self.lock_periods)
+                sampled_addresses = self.alice.recruit(quantity=actual_sample_quantity,
+                                                       duration=self.lock_periods,
+                                                       additional_ursulas=1.0)
```

Comment:
`additional_ursulas=ADDITIONAL_URSULAS`?

Reply:
Thanks for mentioning this! I've been wanting to fix this for a long time but I forgot. There are two places where we were increasing the number of nodes prior sampling: one was here, and another in `StakingEscrow.sample()`. The result was that our default parameters required ~3 times more Ursulas than necessary. I'll reduce this to just one place, governed by the `self.selection_buffer` parameter.